deploy/cr.yaml

```diff
@@ -43,7 +43,7 @@ spec:
 #  userLabels:
 #    pgo-version: "1.5.0"
   pgPrimary:
-    image: perconalab/percona-postgresql-operator:main-ppg14-postgres-ha
+    image: perconalab/percona-postgresql-operator:main-ppg15-postgres-ha
```

Comment:
We need to change these to the ones which will be for the release, like: `percona/percona-postgresql-operator:1.5.0-ppg14-postgres-ha`.
You can have a look at the list of official images in the doc page: https://docs.percona.com/percona-operator-for-postgresql/1.0/images.html
Same for all of the below images in cr.yaml
Also we need to change `percona/pmm-client:2.36.0` to the current latest version.